express/src/views/service-details/invitation-sent.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "layout/base.njk" %}
+
+{% set sidebarActiveItem = "team-members" %}
+{% set pageTitle = "Invitation email sent" %}
+
+{% block beforeContent %}
+  <div class="govuk-phase-banner govuk-clearfix">
+    <div class="govuk-breadcrumbs">
+      <ol class="govuk-breadcrumbs__list navigation-service-change-list">
+        <li class="govuk-breadcrumbs__list-item govuk-!-font-size-19">
+          <a class="govuk-breadcrumbs__link" href="/account/list-services">Your services</a>
+        </li>
+        {% if serviceName %}
+          <li class="govuk-breadcrumbs__list-item govuk-!-font-size-19 govuk-!-font-weight-bold">{{ serviceName }}</li>
+        {% endif %}
+      </ol>
+      <a href="/change-service-name/{{ serviceId }}/{{ selfServiceClientId }}/{{ clientId }}?serviceName={{ serviceName }}"
+         class="govuk-link govuk-link--no-visited-state navigation-service-change govuk-!-font-size-19">
+        Change service name
+      </a>
+    </div>
+  </div>
+{% endblock %}
```

Comment:
Introduce a variable to the `layout-service-details.njk` template to hide the sidebar if needed and avoid duplicating the "Your services > ..." heading

Reply:
Updated.